jsonnet/manifests/prometheus-rules.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
I feel this is the file which is generated and checked in as part of every PR. Right?

Reply:
No. This is what will be generated here during build.